html/archive_body.html

```diff
@@ -0,0 +1,75 @@
+<div id="list-item-__acount__" class="archive-body">
+                        <div>
+                            <h4 class="archive-header">
+                                Build #__acount__
+                            </h4>
+                            <div class="archive-date">
+                                <i class="fa fa-calendar-check-o" aria-hidden="true"></i>&nbsp;&nbsp;&nbsp;
+                                __date__
+                            </div>
+                        </div>
+                        <div style="margin-top: -5%;">
+                            <div id="archive-container-__iloop__" style="padding-top: 5%; position: absolute;">
+                                <div style="">
+                                    <span class="total__tests">__total_tests__</span>
+                                </div>
+                                <div id="archive-label-__iloop__">
+                                    <span class="archive__label">TEST CASES</span>
+                                </div>
+                            </div>
+                            <div class="archive-chart-container">
+                                <canvas id="archive-chart-__iloop__" style="margin-top: 10%; padding-left: 25%; margin-right: -16%; float: right;"></canvas>
+                            </div>
+                        </div>
+                        <div class="archive__bar">
+                            <section id="statistic" class="statistic-section-__status__ one-page-section">
+                                <div class="container" style="margin-top: -2%;">
+                                    <div class="row text-center">
+                                        <div class="col-xs-12 col-md-3" style="max-width: 14.2%;">
+                                            <div class="counter">
+                                                <h2 class="timer count-title count-number">__pass__</h2>
+                                                <p class="stats-text">PASSED</p>
+                                            </div>
+                                        </div>
+                                        <div class="col-xs-12 col-md-3" style="max-width: 14.2%;">
+                                            <div class="counter">
+                                                <h2 class="timer count-title count-number">__fail__
+                                                </h2>
+                                                <p class="stats-text">FAILED</p>
+                                            </div>
+                                        </div>
```

Comment:
indentation

Reply:
Could you please let me know what do you mean under _identation_ ?

Reply:
@kozakHolota shifting the html code to left with better alignment

Reply:
Custom title is something important to fix and apart from that we are good